src/graphql/types/SensitiveString.ts

```diff
@@ -0,0 +1,29 @@
+import { hash } from 'bcrypt';
+import { Kind } from 'graphql';
+import { scalarType } from 'nexus';
+import nullthrows from 'nullthrows';
+
+const { BCRYPT_SALT_ROUNDS: bcryptSaltRounds } = process.env;
+const BCRYPT_SALT_ROUNDS = parseInt(nullthrows(bcryptSaltRounds));
+
+const SensitiveString = scalarType({
```

Comment:
Consider adding short documentation here explaining why a scalarType is used, and how parseValue, parseLiteral, and serialize are for (mostly since half the team isn't quite familiar with graphQL.)

Reply:
I think that's outside of the scope of our code. However, I do think it's a good idea to document this for prying eyes, so I'll leave these here:

https://graphql.org/graphql-js/type/#graphqlscalartype
https://stackoverflow.com/questions/41510880/whats-the-difference-between-parsevalue-and-parseliteral-in-graphqlscalartype